proto/storage/proto.lock

```diff
@@ -17803,4 +17838,4 @@
       }
     }
   ]
-}
+}
```

Comment:
odd that the new line at the end was removed.  that may cause style issues.

Reply:
It is odd, but the autogeneration is removing the line. Don't see any style checks failing on the PR though.